### ISSUE TITLE
Consider addressing lighthouse test issues

### ISSUE THREAD

**Issue by cdmo** [psu-libraries/psulib_blacklight#517]
Some problems to address here, nothing critical:
 
* [lighthouse-desktop-search-results.pdf](https://github.com/psu-libraries/psulib_blacklight/files/3853084/lighthouse-desktop-search-results.pdf)
* [lighthouse-desktop-homepage.pdf](https://github.com/psu-libraries/psulib_blacklight/files/3853085/lighthouse-desktop-homepage.pdf)
* [lighthouse_report.pdf](https://github.com/psu-libraries/psulib_blacklight/files/3853071/lighthouse_report.pdf)
